<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>1400</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QFrame" name="functionFrame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>631</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="showWidget" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>1300</width>
     <height>520</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="titlelabel_1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>数据云平台智能监控系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="titlelabel_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>291</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Data intelligent monitoring system</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="leftWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>100</width>
     <height>520</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
